cranelift/filetests/filetests/runtests/simd-arithmetic-nondeterministic.clif

```diff
@@ -0,0 +1,23 @@
+; Test the non-deterministic aspects of the SIMD arithmetic operations.
+; If you change this file, you should most likely update
+; simd-arithmetic-nondeterministic2.clif as well.
+test run
+target aarch64
+
+function %fmax_f64x2(f64x2, f64x2) -> f64x2 {
+block0(v0: f64x2, v1: f64x2):
+    v2 = fmax v0, v1
+    return v2
+}
+
+; run: %fmax_f64x2([NaN:0x42 0.0], [0x1.0 0.0]) == [NaN:0x42 0.0]
+
+function %fmin_f64x2(f64x2, f64x2) -> f64x2 {
+block0(v0: f64x2, v1: f64x2):
+    v2 = fmin v0, v1
+    return v2
+}
+
+; run: %fmin_f64x2([-NaN 0x100.0], [0.0 NaN]) == [-NaN NaN]
+; run: %fmin_f64x2([NaN 0.0], [0.0 0.0]) == [NaN 0.0]
+; run: %fmin_f64x2([NaN:0x42 0.0], [0x1.0 0.0]) == [NaN:0x42 0.0]
```

Comment:
This pair of tests should probably go in https://github.com/bytecodealliance/wasmtime/tree/main/cranelift/filetests/filetests/isa?

Reply:
More than one target may have the same result for this test. It would be nice to not duplicate it in that case.

Reply:
But that isn't the case here, right?

Reply:
Not yet, but it may in the future as new targets get added. Eg s390x may gove the same result as either x86_64 or aarch64.

Reply:
What @bjorn3 has said is one reason I have kept the files in their present location; there is also an incomplete backend targeting the 32-bit Arm architecture, in which case the behaviour may very well match the AArch64 one (but not necessarily - there are differences between the floating-point handling in AArch32 and AArch64). The other reason is to preserve the current organization of having all runtests in one place, and some of the latter are already meant for a specific backend - the ones with `legacy` in their name (e.g. `cranelift/filetests/filetests/runtests/i128-arithmetic-legacy.clif`), though admittedly they should be removed eventually.

Reply:
Agreed that these tests could be shared. That said, would it be possible to name the file in a way that denotes the behavior a little more specifically? `-nondeterminstic` is good (it indicates that the specified behavior isn't the only valid behavior) but maybe something like `-nondeterministic-armlike` or somesuch (and we can rename if we later add e.g. `s390x`)?